setup.cfg

```diff
@@ -1,35 +1,86 @@
-[egg_info]
-#tag_build = .dev
-#tag_svn_revision = 1
+[metadata]
+name = yt
+version = 4.0.dev0
+description = An analysis and visualization toolkit for volumetric data
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/yt-project/yt
+author = The yt project
+author_email = yt-dev@python.org
+license = BSD 3-Clause
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Framework :: Matplotlib
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: MacOS :: MacOS X
+    Operating System :: POSIX :: AIX
+    Operating System :: POSIX :: Linux
+    Programming Language :: C
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering :: Astronomy
+    Topic :: Scientific/Engineering :: Physics
+    Topic :: Scientific/Engineering :: Visualization
+keywords = astronomy astrophysics visualization amr adaptivemeshrefinement
+project_urls =
+    Homepage=https://yt-project.org/
+    Documentation=https://yt-project.org/doc/
+    Source=https://github.com/yt-project/yt/
+    Tracker=https://github.com/yt-project/yt/issues
 
+[options]
+packages = find:
```

Comment:
Wouldn't this fit instead in `pyproject.toml`?

Reply:
I don't think so.

Reply:
For reference, these two PEPs are ongoing
https://www.python.org/dev/peps/pep-0621/
https://www.python.org/dev/peps/pep-0631/